_data/events.yml

```diff
@@ -93,313 +93,6 @@
   gh_labels: code-along
 
 
-   # Key is the part of the filename in _posts/
-- key: inkscape
```

Comment:
We'll obviously want to put up the first 4 events that we talked about + the coffee and code sessions. Could do this in a separate PR though.